documentation/explainers/synthetic_tokens.md

```diff
@@ -0,0 +1,86 @@
+
+This tutorial will show you how to mint tokens from the command line after having parameterized and deployed the synthetic token’s smart contract.
+
+## Managing Token Sponsor Positions
+During the lifetime of the synthetic token, token sponsors may want to deposit additional collateral to their positions so as to avoid liquidation. Sponsors may also want to withdraw excess collateral if the position has moved in their favor. 
+
+Token sponsors can deposit additional collateral at any time.
+
+Token sponsors can withdraw excess collateral in one of two ways: a “fast” withdrawal or “slow” withdrawal.
+
+### “Fast” withdrawal:
+
+
+A “fast” withdrawal allows a token sponsor to withdraw excess collateral from his position immediately, so long as the resulting position is collateralized by at least as much as the average token sponsor collateralization ratio for the contract (“global collateralization ratio”). [“global collateralization ratio” is a link to a glossary of terms.]
+ 
+### “Slow” withdrawal: 
+
+If the token sponsor wishes to withdraw collateral from his position that would bring his collateralization below the global collateralization ratio, he can do so via a “slow” withdrawal. Because withdrawing this amount of collateral could potentially jeopardize the solvency of the fungible synthetic tokens, this “slow”, 2-part, withdrawal process allows other token holders to flag if a withdrawal would render the token sponsor insolvent. 
+
+In a “slow” withdrawal, there are two parts: 
+The token sponsor submits a withdrawal request to the contract indicating the amount of collateral he wishes to withdraw and the timestamp of the request.
+
+During this period, any token holder can liquidate the token sponsor’s position if they believe a withdrawal of the amount indicated in the withdrawal request would bring the token sponsor’s collateralization below the “collateralization requirement” at the time of liquidation. [“collateralization requirement” is a link to a glossary of terms.] 
+If the “withdrawal liveness period” passes without a token holder liquidating the token sponsor, the token sponsor may withdraw collateral from his position up to the amount requested. [“withdrawal liveness period” is a link to a glossary of terms.] 
+
+## Liquidation and Dispute
+At any time, a token holder may liquidate a token sponsor’s position. Liquidations happen immediately without calling the oracle. Anyone may dispute a liquidation within the “liquidation liveness period”. [“withdrawal liveness period” is a link to a glossary of terms.] 
```

Comment:
Because they are two separate parameters in the parameterization of a contract for a new synthetic token, I would call them "withdrawal liveness period" and "liquidation liveness period".